dataland-frontend/src/components/forms/CreateEuTaxonomyForFinancials.vue

```diff
@@ -559,7 +561,11 @@ export default defineComponent({
           this.formInputsModel.data as ObjectType,
           this.namesOfAllCompanyReportsForTheDataset
         );
-        await (this.$refs.UploadReports.uploadFiles as () => Promise<void>)();
+
```

Comment:
Was this changed due to the linter/sonar finding?

Reply:
This was changed in the context of refactoring and reducing code duplication for file uploads through the frontend. So, yes for sonar but preemptively.

Reply:
ok